bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -226,6 +226,20 @@ export default defineComponent({
 						],
 					},
 				},
+				{
+					name: "Local Extrema",
+					callback: (canvas: HTMLCanvasElement, bezier: WasmBezierInstance): void => {
+						const context = getContextFromCanvas(canvas);
+						const dimensionColours = [COLORS.NON_INTERACTIVE.STROKE_1, COLORS.NON_INTERACTIVE.STROKE_2];
```

Comment:
Please spell it `Colors` since we have to be consistent and we arbitrarily decided on American English (well, since I'm American, but also because the Rust project uses American English).